deploy/cloudformation/elastic-agent-ec2.yml

```diff
@@ -1,10 +1,10 @@
 ---
-AWSTemplateFormatVersion: '2010-09-09'
+AWSTemplateFormatVersion: "2010-09-09"
 Description: Creates an EC2 instance with permissions to run elastic-agent
 Parameters:
   LatestAmiId:
     Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
-    Default: '/aws/service/canonical/ubuntu/server/jammy/stable/current/arm64/hvm/ebs-gp2/ami-id'
+    Default: '/aws/service/ami-amazon-linux-latest/al2023-ami-minimal-kernel-default-arm64'
```

Comment:
> AWS provides Amazon Linux 2023 and Amazon Linux 2. If you are migrating from another Linux distribution to Amazon Linux, we recommend that you migrate to Amazon Linux 2023.

[source](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/amazon-linux-ami-basics.html#:~:text=AWS%20provides%20Amazon%20Linux%202023%20and%20Amazon%20Linux%202.%20If%20you%20are%20migrating%20from%20another%20Linux%20distribution%20to%20Amazon%20Linux%2C%20we%20recommend%20that%20you%20migrate%20to%20Amazon%20Linux%202023.)

seems like the recommended image. also both images - [amazon linux 2](https://aws.amazon.com/amazon-linux-2/?amazon-linux-whats-new.sort-by=item.additionalFields.postDateTime&amazon-linux-whats-new.sort-order=desc) and [amazon linux 2023](https://aws.amazon.com/linux/amazon-linux-2023/) have the same scan results